charts/timescaledb-single/docs/admin-guide.md

```diff
@@ -78,7 +78,7 @@ The following table lists the configurable parameters of the TimescaleDB Helm ch
 | `podMonitor.namespace`        | Setting this will cause deploying podMonitor in a different namespace than TimescaleDB. | `nil` |
 | `podMonitor.labels`           | Additional labels that can be set on podMonitor object. | `nil` |
 | `podMonitor.metricRelabelings` | Additional prometheus metric relabelings. | `nil` |
-| `podMonitor.targetLabels`     | List of additional kubernetes labels that need to be transferred from Service object into metrics. | `nil` |
+| `podMonitor.targetLabels`     | List of additional kubernetes labels that need to be transferred from Pod object into metrics. | `nil` |
```

Comment:
To keep confusion down lets change this to match what is in the PodMonitor spec as well, `podMonitor.podTargetLabels`

Reply:
Thx for checking. Will do so.